src/foremast/app/create_app.py

```diff
@@ -85,13 +86,22 @@ def create_app(self, pipeline_configs=None):
         Raises:
             AssertionError: Application creation failed.
         """
+        self.pipeline_configs = pipeline_configs
         self.appinfo['accounts'] = self.get_accounts()
         self.log.debug('App info:\n%s', pformat(self.appinfo))
-
+        self.add_to_instance_links()
         jsondata = get_template(template_file='infrastructure/app_data.json.j2',
-                                appinfo=self.appinfo, pipeline_configs=pipeline_configs)
+                                appinfo=self.appinfo, pipeline_configs=self.pipeline_configs)
 
         wait_for_task(jsondata)
 
         self.log.info("Successfully created %s application", self.appname)
         return
+
+    def add_to_instance_links(self):
```

Comment:
I'm always cautious about inline mutations of objects. I would rather prefer this return the calculation/appending of links.

This may need some debug/info output. I'd probably suggest `debug` before mutation and `info` after mutation.